docs/key_portability.md

```diff
@@ -0,0 +1,58 @@
+---
+id: key_portability
+title: Key Portability and Import Considerations
+---
+
+# Trade-offs around Key Portability and Key Import
+
+Two important concepts to consider when managing cryptographic keys are key portability and key import.
+
+Key portability refers to moving keys between different systems or devices. This can be useful for backup, disaster recovery, and migrating to a new system.
+
+On the other hand, key import refers to the ability to securely bring keys into a system. This is necessary when keys are created on a different system or device or provided by a third party.
+
+There are trade-offs to consider when designing a system that must balance key portability and import. Here are a few key considerations:
+
+## Security
+
+Security is always the top priority when it comes to cryptographic keys. A system that prioritizes key portability over key import may sacrifice security to make it easier to move keys between systems. On the other hand, a system that prioritizes key import over key portability may require more secure methods for importing keys, such as secure channels or physical transfer.
+
+## Compatibility
+
+When it comes to key portability, compatibility is an important consideration. Keys should be compatible between different systems and devices and should be stored in a format that is widely recognized and accepted. Key import may require additional steps to ensure compatibility, such as converting keys to a different format or ensuring that the importing system supports the same cryptographic algorithms as the exporting system.
+
+## Complexity
+
+Key portability can add complexity to a system, as it may require additional steps and processes to ensure that keys are properly moved between systems. Key import can also add complexity, as it may require additional security measures and validation steps to ensure the imported keys are legitimate.
+
+## Use Cases
+
+It's important to consider the specific use cases for key portability and key import. In some cases, key portability may be more important, such as in disaster recovery scenarios where the ability to quickly recover keys is critical. In other cases, key import may be more important, such as when working with third-party systems or when importing keys for a specific purpose.
+
+# Examples of Trade-offs between Key Portability and Key Import
```

Comment:
I think this page is misguided.  The page is discussing the trade offs between key portability and key import.  As I understand it, key import is a feature that _supports_ key portability.  The trade off is key portability and key security.  At least, this is how I understand it. So, to me, I have trouble approving this page.

Reply:
Rewritten to take this observation into account.